README.md

```diff
@@ -17,6 +17,58 @@ pip install -r requirements.txt
 
 ## Example
 
+### Compute SNARK-friendly Pedersen hash
+Let's create a simple demo, called `demo.py`:
+```python
+from zokrates.gadgets.pedersenHasher import PedersenHasher
+
+preimage = bytes.fromhex("1616")
+# create a instance with personalisation string
+hasher = PedersenHasher(b"test")
+# hash payload
+digest = hasher.hash_bytes(preimage)
+print(digest)
+# x:2685288813799964008676827085163841323150845457335242286797566359029072666741,
+# y:3621301112689898657718575625160907319236763714743560759856749092648347440543
+
+# write ZoKrates DSL code to disk
+path = "pedersen.code"
+with open(path, "w+") as f:
+    for l in hasher.dsl_code:
+        f.write(l)
+
+# write witness arguments to disk
+path = "pedersen_witness.txt"
+witness = hasher.gen_dsl_witness_bytes(preimage)
+with open(path, "w+") as f:
+    f.write(" ".join(witness))
```

Comment:
we would need this helper for bits, bytes and scalar or i think we dont gain that much. any recommendation for a more syntax efficient wrapper?

Reply:
I see. I guess there are a couple of ways to do it:
* one would be to define a `gen_dsl_witness` method, which internally checks the type of input using `instanceof` (or whatever other check to ensure the input is one of the correct types) and then proceeds accordingly
* an alternative might be to use `**kwargs` and have support for named arguments such as `bits_input`, `bytes_input`, `scalar_input` (the `_input` suffix is just a suggestion since `bytes` is a reserved keyword) and essentially require the user of the library to specify the type in this way. However, if it's done like this, I think it would still require some checks that the input is in the correct format for the given type, so I guess the first option might be more reasonable.